server/plugin.go

```diff
@@ -513,6 +535,25 @@ func (p *Plugin) onActivate() error {
 			return dbErr
 		}
 
+		// simulate a ping to the plugin from server so it appears "online" immediately.
+		// !!!! remove this when fixed in server
+		_, err = db.Exec("update remoteclusters set lastpingat=$1 where remoteid=$2;", model.GetMillis(), p.remoteID)
+		if err != nil {
+			return fmt.Errorf("cannot simulate ping: %w", err)
```

Comment:
Should this fail the plugin startup, or can we continue safely?

Reply:
I guess we could continue safely, but if this is failing, almost 100% sure is because something is really wrong the the DB connection, so I don't see the point on keep going.

Reply:
This is no longer needed.  Covered by https://github.com/mattermost/mattermost/pull/26753